packages/boilerplate-generator/template.js

```diff
@@ -1,4 +1,4 @@
-import lodashTemplate from 'lodash.template';
+import { template as lodashTemplate } from 'lodash';
 
```

Comment:
Is this correct? maybe we should `import { _ }` and user `_.template`

Reply:
no because the _ is the full lodash as we can see in the readme
```
var _ = require('lodash');
```

since we will work only with template, i'm using a descrontructor ` var { template} = require('lodash')`